### ISSUE TITLE
Prevalues fail with no caption

### ISSUE THREAD

**Issue by alandraper** [umbraco/Umbraco.Forms.Issues#1140]
Umbraco Forms 11.2.1

Adding a dropdown list and filling in the Values but not the Captions causes a null reference exception reading the form 

Reproduction
------------

Create a new form; add a dropdown; add some prevalues, but leave the "Captions" blank. 

Save the form and reload the page. 

The form will be unreadable and undeletable. The only way to fix it is to edit the json directly in the database. 

The error is in FieldPrevalueJsonConverter.cs which seems like it needs some null reference checks

```
JToken jToken = JToken.Load(reader);
if (jToken is JObject jObject)
{
	return new FieldPrevalue
	{
		Value = jObject["value"].ToString(),
		Caption = jObject["caption"].ToString() // causes exception
	};
}
```

---
_This item has been added to our backlog AB#36279_

**Comment by AndyButland** [comment]
Strange... I agree the code you have shown looks problematic, but in testing I don't see a problem here.  Even with the `caption` with a value of `null`, the expression comes back as an empty string.  In any case though we can be more defensive here and hopefully that'll sort it out for all cases.

**Comment by alandraper** [comment]
It definitely causes an error. Just came across it again for a different client.

Just update a form definition in the database with the prevalues like this to see the error

```
    "preValues": [
        {
            "value": "1"
        },
        {
            "value": "2"
        },
        {
            "value": "3"
        },
        {
            "value": "4"
        }
    ],

**Comment by AndyButland** [comment]
What version are you on now please @alandraper where you are seeing this problem still?

**Comment by alandraper** [comment]
They're on 11.2.1

**Comment by AndyButland** [comment]
OK, so they won't have the fix I'm afraid - this was resolved in 12.2.2 and 13.0.1 as per the labels on this issue.  Forms 11 (and Umbraco 11) is "end of life" now though, so we aren't planning to make further updates to that version.  Ideally I think you would get them onto Umbraco 13, which is a "long-term support" version.

**Comment by alandraper** [comment]
Yep. Was just showing how you can replicate the error.

And yes, the upgrade will happen as soon as time & budget allow.
